sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeCustomFormsOperation.cs

```diff
@@ -134,5 +143,36 @@ private static IReadOnlyList<RecognizedForm> ConvertSupervisedResult(AnalyzeResu
             }
             return forms;
         }
+
+        private static async ValueTask<RequestFailedException> CreateExceptionForFailedOperationAsync(bool async, ClientDiagnostics diagnostics, Response response, IReadOnlyList<FormRecognizerError> errors)
```

Comment:
Just curious - why make this static and pass in private members, rather than make it non-static and access them directly?

Reply:
Just me being careless. I only had the `bool` and the list when I started writing the method, so it made sense at that point. I think it's cleaner to remove the `static` keyword.